src/main/java/org/springframework/data/repository/query/ExtensionAwareEvaluationContextProvider.java

```diff
@@ -424,25 +424,25 @@ public EvaluationContextExtensionAdapter(EvaluationContextExtension extension,
 		/**
 		 * Returns the extension identifier.
 		 * 
-		 * @return
+		 * @return the id of the extension
 		 */
-		public String getExtensionId() {
+		String getExtensionId() {
 			return extension.getExtensionId();
 		}
 
 		/**
 		 * Returns all functions exposed.
 		 * 
-		 * @return
+		 * @return all exposed functions.
 		 */
-		public Map<String, Function> getFunctions() {
+		FunctionsMap getFunctions() {
```

Comment:
Wonder if it makes sense to extract an interface here so that clients can't modify the instance anymore, i.e. return something that doesn't expose the `addAll(…)` methods anymore.